template_functions.go

```diff
@@ -210,15 +210,15 @@ func nodesFunc(brain *Brain,
 
 // secretFunc returns or accumulates secret dependencies from Vault.
 func secretFunc(brain *Brain,
```

Comment:
Do we need to do the same for `secretsFunc()` below? Not sure how that relates to this one.